File: Easy/clean_up_the_words.py
data = ['(--9Hello----World...--)', 'Can 0$9 ---you~', '13What213are;11you-123+138doing7']


import string
print string.ascii_lowercase
print string.ascii_uppercase

#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	l = len(test)
	sentence = ''
	for c in range(0, l):
		char = test[c]
		if ((char in string.ascii_lowercase) or (char in string.ascii_uppercase)):
			sentence = sentence + char
		else:
			sentence = sentence + " "
	sentence = " ".join(sentence.split())
	print sentence.lower()

#test_cases.close()

File: Easy/clean_up_the_words.r
test.cases <- c("(--9Hello----World...--)", "Can 0$9 ---you~", "13What213are;11you-123+138doing7")

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		words_with_spaces <- gsub("[^[:alpha:]]", " ", test)
		just_words <- gsub("^ *|(?<= ) | *$", "", words_with_spaces, perl = TRUE)
		lower_words <- tolower(just_words)
	}
}), sep = "\n")